### PR TITLE
add github action to build and push docker images

### PR DESCRIPTION
**Pull request by mad73923**
- your script folder may be deleted!?
- to enable pushing to docker hub, uncomment the lines in the action and provide the secrets in your settings.

cheers

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# https://github.com/DRuggeri/alertmanager_gotify_bridge/issues/12
```
Comment (ngosang) [inline Dockerfile]: I think you don't need the comment neither the ARCH arg.

Comment (mad73923) [reply Dockerfile]: thanks. done

**Comment by DRuggeri** [comment]
Thanks, guys! This is a great start!
I ended up having issues with a few minor things, but will follow up with a fixing PR in just a moment!

**Comment by mad73923** [comment]
Hey @DRuggeri ,
thanks for the integration!
somehow, some platforms got lost on the way. Can you revert this?

**Comment by DRuggeri** [comment]
Hi, @mad73923 - I'm not sure what you mean by this? The release actually now contains several more platforms than before.

Added:
* freebsd-386
* freebsd-amd64
* freebsd-arm
* freebsd-arm64
* windows-arm64
* windows-arm

Because performing a release is a bit of a chore for me, I don't intend to revert the pipeline-based flow... but I'm happy to fix it forward if something's broken. Can you toss details in a new issue to clarify what seems to be the problem?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64
```
Comment (mad73923) [inline .github/workflows/ci.yml:52]: @DRuggeri These are the platforms which were supported originally. Would be nice to integrate them to be able to run this on SBCs like RaspPi

Comment (DRuggeri) [reply .github/workflows/ci.yml:52]: I see now... but is there a practical difference between that list of architectures and "linux/arm" + "linux/arm64"? The golang binary is the same (it only differentiates between "arm" and "arm64") and it's a scratch-based container so there are no other architecture-related files to be concerned with.

Have you tested on a Pi with Docker? I'm not at a spot to do that right now, but it ought to be functional as it is

Comment (mad73923) [reply .github/workflows/ci.yml:52]: You are right! The container is running smoothly on my RockPi.
Thanks for your help!

**Pull request merged**